threedi_schema/migrations/versions/0222_upgrade_db_settings.py

```diff
@@ -303,7 +325,7 @@ def upgrade():
     unused_cols = [settings_id for _, settings_id, _ in GLOBAL_SETTINGS_ID_TO_BOOL] + ["numerical_settings_id"]
     drop_columns('model_settings', unused_cols)
     # remove relative path prefix from dem path
```

Comment:
from raster paths

Reply:
Resolved in 409c4b2c86f29e13cce0f0028706428d664eaafa